### PR TITLE
Turn on white header/footer theme. Hide date option on MG setup for a…

### PR DESCRIPTION
**Pull request by mcstover**
…ll depositors coming from covid landing page.

### CODE REVIEW

File: src/pages/LandingPages/MGCovid19/MGCovid19.vue [hunk 1]
```diff
@@ -110,6 +114,20 @@ export default {
 	},
 	data() {
 		return {
+			co19HeaderTheme: {
+				themeKey: 'CO19',
```
Comment (ryan-ludwig) [inline src/pages/LandingPages/MGCovid19/MGCovid19.vue:118]: What does themeKey do?

File: src/pages/LandingPages/MGCovid19/MGCovid19.vue [hunk 2]
```diff
@@ -162,7 +180,7 @@ export default {
 }
 
 .mg-hero {
-	background-color: $kiva-green;
+	background-color: $white;
```
Comment (ryan-ludwig) [inline src/pages/LandingPages/MGCovid19/MGCovid19.vue:183]: I did notice that when you resize now, the white background shows below the image.

File: src/pages/LandingPages/MGCovid19/MGCovid19.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/LandingPages/MGCovid19/MGCovid19.vue:118]: It's used for the cache in the footer. It' snot used for caching in the header as we don't cache that component.

File: src/pages/LandingPages/MGCovid19/MGCovid19.vue (hunk 2 above)
Comment (mcstover) [reply src/pages/LandingPages/MGCovid19/MGCovid19.vue:183]: Yea the new design updates are all on a white bg so I removed the green in preparation...

**Pull request merged**